gammapy/spectrum/fit.py

```diff
@@ -19,108 +18,44 @@ class SpectrumFit(object):
     """
     Spectral Fit using Sherpa
 
-    Disclaimer: Gammapy classes cannot be translated to Sherpa classes yet.
-    Therefore the input data must have been written to disk in order to be read
-    in directly by Sherpa.
+    This is a wrapper around `~sherpa.fit.Fit` that takes care about
+    translating gammapy classes to sherpa classes and handling various aspects
+    of the fitting correctly. For usage examples see :ref:`spectral_fitting`
 
     Parameters
     ----------
-    obs_list : SpectrumObservationList
+    obs : SpectrumObservationList
         Observations to fit
-
-    Examples
-    --------
-
-    Example how to run a spectral analysis and have a quick look at the results.
-
-    ::
-
-        from gammapy.spectrum import SpectrumObservation, SpectrumFit
-        import matplotlib.pyplot as plt
-
-        filename = '$GAMMAPY_EXTRA/datasets/hess-crab4_pha/pha_obs23523.fits'
-        obs = SpectrumObservation.read(filename)
-
-        fit = SpectrumFit(obs)
-        fit.run()
-        fit.result.plot_fit()
-        plt.show()
-
-    TODO: put output image in gammapy-extra and show it here.
+    model : `~gammapy.spectrum.models.SpectralModel`, `~sherpa.models.ArithmeticModel`
+        Model to be fit
+    stat : str, `~sherpa.stats.Stat` 
+        Fit statistic to be used
     """
-
     FLUX_FACTOR = 1e-20
     """Numerical constant to bring Sherpa optimizers in valid range"""
     DEFAULT_STAT = 'wstat'
     """Default statistic to be used for the fit"""
-    DEFAULT_MODEL = models.PowerLaw(index=2 * u.Unit(''),
-                                    amplitude=1e-11 * u.Unit('cm-2 s-1 TeV-1'),
-                                    reference=1 * u.TeV)
-    """Default model to be used for the fit"""
-
-    def __init__(self, obs_list, stat=DEFAULT_STAT, model=DEFAULT_MODEL):
-        if isinstance(obs_list, SpectrumObservation):
-            obs_list = SpectrumObservationList([obs_list])
-        if not isinstance(obs_list, SpectrumObservationList):
-            raise ValueError('Wrong input format {}\nUse SpectrumObservation'
-                             'List'.format(type(obs_list)))
-
-        self.obs_list = obs_list
+
+    def __init__(self, obs, model, stat=DEFAULT_STAT):
+        if isinstance(obs, SpectrumObservation):
```

Comment:
I'm -1 on doing `obs = [obs]` automatically and to accept both list of obs and obs.
If you just accept list of obs callers can with two extra characters make a list and pass that in.
The advantage of just accepting one thing is simpler API / documentation and variable names that match better (should be `obs_list` here).

But if you really like this as a convenience here, just ignore me, no need to argue.